grails-app/services/au/org/ala/spatial/portal/HubWebService.groovy

```diff
@@ -78,17 +78,27 @@ class HubWebService {
 
         try {
             if (type == HttpPost.METHOD_NAME) {
-                call = new PostMethod()
+                call = new PostMethod(url)
                 if (inputStream) {
                     call.setRequestEntity(new InputStreamRequestEntity(inputStream))
                 }
             } else {
```

Comment:
else not necessarily is Get, so better use `else if (type == HttpGet.METHOD_GET)` ?

Reply:
The method getStream only can be called by Post/Get, so it is OK now.